lib/govuk_publishing_components/presenters/attachment.rb

```diff
@@ -48,6 +50,14 @@ def readable_number_of_pages
         pluralize(attachment_data[:number_of_pages], "page")
       end
 
+      def thumbnail
+        "https://www.gov.uk/government/assets/pub-cover-doc-afe3b0a8a9511beeca890340170aee8b5649413f948e512c9b8ce432d8513d32.png"
+      end
+
+      def opendocument?
+        OPENDOCUMENT_ABBREVIATIONS.include?(content_type_abbr)
```

Comment:
I don't think we should couple this to abbreviation as that is a frontend rendering concern. 

What I'd suggest we do is set constants for the mime types and then replace their string usage in https://github.com/alphagov/govuk_publishing_components/blob/57ff9157c9c8be3902acb8fc154265ed35464c29/lib/govuk_publishing_components/presenters/attachment.rb#L73-L75

and then check the content_type is one of those.

Reply:
As discussed, `#opendocument?` now delegates to SupportedContentType to check if it's an OpenDocument or not.